### PR TITLE
Localisation

### PR DESCRIPTION
**Pull request by IrinaBukley23**

### CODE REVIEW

File: my-app/src/components/tasksBlock/tasksBlock.tsx [hunk 1]
```diff
@@ -46,7 +47,7 @@ const TasksBlock = () => {
   return (
     <Stack spacing={2} className={styles.central}>
       <TextField
-        placeholder='Начните вводить для поиска...'
+        placeholder={t('taskSearch')}
```
Comment (sunnyfur) [inline my-app/src/components/tasksBlock/tasksBlock.tsx:50]: This line generate an error in my application: Type 'DefaultTFuncReturn' is not assignable to type 'string | undefined'.
  Type 'null' is not assignable to type 'string | undefined'

File: my-app/src/components/tasksBlock/tasksBlock.tsx [hunk 2]
```diff
@@ -58,7 +59,7 @@ const TasksBlock = () => {
         onChange={handleSearchChange}
       />
       <TextField
-        placeholder='Введите задачу'
+        placeholder={t('taskCreatePlaceholder')}
```
Comment (sunnyfur) [inline my-app/src/components/tasksBlock/tasksBlock.tsx:62]: The same problem: Type 'DefaultTFuncReturn' is not assignable to type 'string | undefined'.
Type 'null' is not assignable to type 'string | undefined'

File: my-app/src/components/sideBar/sideBar.tsx [hunk 3]
```diff
@@ -42,6 +44,7 @@ const SideBar = () => {
       dispatch(setCurrDate(moment(date).format('YYYY-MM-DD HH:mm')))
     }
   }
+  const { t, } = useTranslation();
```
Comment (sunnyfur) [inline my-app/src/components/sideBar/sideBar.tsx:52]: Does we need comma here?

File: my-app/src/components/sideBar/sideBar.tsx [hunk 4]
```diff
@@ -91,18 +94,18 @@ const SideBar = () => {
           aria-controls='panel1a-content'
           id='panel1a-header'
```
Comment (sunnyfur) [inline my-app/src/components/sideBar/sideBar.tsx:99]: In localizationProvider switch adapterLocale to ru/en for localization

File: my-app/src/components/tasksBlock/taskCard.tsx [hunk 5]
```diff
@@ -167,7 +172,7 @@ const TaskCard = ({ task, onDelete, onChange }: Props) => {
             <LocalizationProvider dateAdapter={AdapterMoment} adapterLocale='ru'>
```
Comment (sunnyfur) [inline my-app/src/components/tasksBlock/taskCard.tsx]: Switch adapterLocale to en/ru for localization

**Review by sunnyfur — APPROVED** [review]

File: my-app/src/components/tasksBlock/tasksBlock.tsx (hunk 1 above)
Comment (IrinaBukley23) [reply my-app/src/components/tasksBlock/tasksBlock.tsx:50]: done

File: my-app/src/components/tasksBlock/tasksBlock.tsx (hunk 2 above)
Comment (IrinaBukley23) [reply my-app/src/components/tasksBlock/tasksBlock.tsx:62]: done

File: my-app/src/components/sideBar/sideBar.tsx (hunk 3 above)
Comment (IrinaBukley23) [reply my-app/src/components/sideBar/sideBar.tsx:52]: corrected

File: my-app/src/components/tasksBlock/taskCard.tsx (hunk 5 above)
Comment (IrinaBukley23) [reply my-app/src/components/tasksBlock/taskCard.tsx]: done

**Pull request merged**